Sharpmake.Generators/VisualStudio/Csproj.cs

```diff
@@ -2288,30 +2288,18 @@ List<string> skipFiles
                                         ? "TextTemplatingFilePreprocessor"
                                         : "TextTemplatingFileGenerator";
 
-                //Add the generated file if its in the remaining files.
-                bool generatedFileFound = remainingSourcesFiles.Concat(remainingResourcesFiles)
-                    .Concat(remainingEmbeddedResourcesFiles)
-                    .Concat(remainingNoneFiles).Contains(generatedFile);
-                AddContentGeneratedItem(itemGroups, ttFile, generatedFile, generator, false, _projectPathCapitalized, project, generatedFileFound);
-
+                // Always include files generated by text templating (.tt), even if they don't exist yet.
+                // These files are expected to be absent during a clean build, as they are generated
+                // during the build process. Therefore, they cannot be left out of the csproj file.
+                AddContentGeneratedItem(itemGroups, ttFile, generatedFile, generator, false, _projectPathCapitalized, project, true);
```

Comment:
AddContentGeneratedItem is only use here. we should remove the last parameter.

Reply:
Removed the last parameter. Sorry for the long delay. Thanks for your attention!